tests/lib/sample_rate_converter/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
```

Comment:
I think this is enabled by default.

Reply:
Seems that it actually isn't, it is disabled when I remove it from the .conf file, and the definition doesn't have a default.

Reply:
Yes. Rebase on main now as the upmerge is done and remove this.

Reply:
That seems to be correct - removed